pentaho/impl/src/test/java/pt/webdetails/cdf/dd/settings/CdfDDSettingsForTesting.java

```diff
@@ -40,7 +40,7 @@ public CdfDDSettingsForTesting( IRWAccess writeAccess ) {
   }
 
   @Override
-  protected List<Element> getSettingsXmlSection( String section ) {
+  protected List<Node> getSettingsXmlSection( String section ) {
```

Comment:
Changing this method to `List<Node>` forced a lot of the calling call to change the declared list type, without need.
Just rename the method to `getSettingsXmlSectionElements`, to make it clearer, and downcast the result of `selectNodes`.